README.md

```diff
@@ -100,7 +100,7 @@ end
 | Setting           | Default | Values          | Description
 |:------------------|:--------|-----------------|------------
 | `adapter`         | `:nokogiri` | `:nokogiri`, `:ox` | The XML parser used by Saxerator |
-| `output_type`     | `:hash` | `:hash`, `:xml` | The type of object generated by Saxerator's parsing. `:hash` should be self-explanatory, `:xml` generates a `Nokogiri::XML::Document`
+| `output_type`     | `:hash` | `:hash`, `:xml` | The type of object generated by Saxerator's parsing. `:hash` should be self-explanatory, `:xml` generates a `REXML::Document` (but have additional method :to_xml)
```

Comment:
Removing our REXML::Document wrapper means we can remove the documentation of `to_xml` here